bin/send_lhe.py

```diff
@@ -95,16 +95,32 @@ def send(self):
             frun.write('cd job%s_%s\n'%(uid,self.process))
             frun.write('export EOS_MGM_URL=\"root://eospublic.cern.ch\"\n')
             frun.write('source %s\n'%(self.para.defaultstack))
-            frun.write('source /cvmfs/sft.cern.ch/lcg/views/LCG_97a_FCC_4/x86_64-centos7-gcc8-opt/setup.sh \n')
+            # frun.write('source /cvmfs/sft.cern.ch/lcg/views/LCG_97a_FCC_4/x86_64-centos7-gcc8-opt/setup.sh \n') #setting this release up clashes with defaultstack setup w.r.t python version, causing runtime errors -> removed. what is it needed for?
```

Comment:
Some packages may have been missing from older stacks. Also, compiler may have been too old on the centos7.